elastic/kibana
# [Ingest Manager][Endpoint] Add Endpoint Create Policy flow with Ingest

## Summary

Creates flow that will allow a user to click a "Create" button in Endpoint, be redirected to Ingest create datasource screen (Integrations > {package-version} > create data source) and once that is complete, have the user navigate back to Endpoint.

Closes #68914

![emt-385-create-policy-flow](https://user-images.githubusercontent.com/56442535/84680780-39eec980-af01-11ea-9dfd-108cd483280a.gif)



#### How to Test

1. Navigate to Endpoint Policy page (`/app/security#/management/policy`)
2. Click the `Create new policy` button at the top-right hand corner.
Page will navigate to Ingest's Integrations section, and load the Endpoint Create Datasource page
3. Select the Default Agent Config and Click `Save`.
After successfully saving the datasource, you should be redirected back to the Endpoint policy list page.


### Checklist

Delete any items that are not applicable to this PR.

A few other manual tests that were done:
  1. Hovering over the Cancel Button or Cancel back link will show the Endpoint URL instead of the Ingest related URLs
  2. After being redirected from Endpoint to Ingest, if user clicks one of the other sections (ex. Configurations) and then clicks back to Integrations and follows the flow for a new datasource, the behaviors for the Cancel Link/Button and successful Save should be that of a normal ingest flow
